lib/client/keyagent.go

```diff
@@ -36,6 +36,13 @@ type LocalKeyAgent struct {
 	agent.Agent               // Agent is the teleport agent
 	keyStore    LocalKeyStore // keyStore is the storage backend for certificates and keys
 	sshAgent    agent.Agent   // sshAgent is the system ssh agent
+
+	// map of "no hosts". these are hosts that user manually (via keyboard
+	// input) refused connecting to.
+	noHosts map[string]bool
```

Comment:
When would `noHosts` be used? It doesn't appear to be persisted to disk and isn't `LocalKeyAgent` recreated everytime you run `tsh`?